orttraining/orttraining/python/training/ortmodule/_io.py

```diff
@@ -113,21 +135,15 @@ def __repr__(self) -> str:
             \tRequire gradient:                 {self.require_grad_names}
             \tDynamic axes:                     {self.dynamic_axes}
             \tSchema:                           {self.schema}
-            \t#Positionals (total):             {self.num_positionals}
-            \t#Expanded Positionals (non-None): {self.num_expanded_positionals_non_none}
-            \tKeyword names:                    {self.keyword_names}"""
```

Comment:
By not printing the captured names, we make debugging of scenarios with kwargs harder

Reply:
I agree. Let me merge this change to unblock a model, and I will create a follow up PR to add the flattened keyword names as part of the `InputInfo`.